src/test/java/com/github/invictum/test/dto/DtoInheritanceTest.java

```diff
@@ -0,0 +1,50 @@
+package com.github.invictum.test.dto;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@RunWith(JUnit4.class)
+public class DtoInheritanceTest {
+
+    private ExampleInheritedDTO sut = null;
+
+    @Before
+    public void initTestDto() {
+        sut = new ExampleInheritedDTO();
+    }
+
+    @Test
+    public void mainDtoTest() {
+        sut.setText1("text1");
+        sut.setText2("text2");
+        String expected = "{text1: text1, text2: text2, intValue: 0}";
```

Comment:
As you can see here are unusual case with int, it's because int cann't be null. We can leave as is or ignore int if it value equal 0. What do you think?